app/services/notifications/new_comment/send.rb

```diff
@@ -13,8 +13,11 @@ def self.call(*args)
       end
 
       def call
-        user_ids = comment.ancestors.select(:user_id).where(receive_notifications: true).pluck(:user_id).to_set
-        user_ids.add(comment.commentable.user_id) if comment.commentable.receive_notifications
+        comment_user_ids = comment.ancestors.select(:user_id).where(receive_notifications: true).pluck(:user_id)
```

Comment:
I don't think you need both select and pluck here, only `pluck` should be enough